### ISSUE TITLE
Line chart not rendering in `0.6.0` or `0.7.0`

### ISSUE THREAD

**Issue by boygirl** [FormidableLabs/victory-native#63]
**From @ryancoughlin on https://github.com/FormidableLabs/victory-native/pull/56**
I pulled down the latest release and my line chart won't render anymore. Previously I only had this issue when running in release mode. It now happens in both environments. Few notes:

- Tried stripping down all my other victory elements so I was only rendering a default `VictoryLine` 
- Fresh install of `victory-native` including `react-native-svg` 

`package.json`
```
  "dependencies": {
    "eslint-plugin-react-native": "^2.2.0",
    "lodash": "^4.16.6",
    "moment": "^2.15.2",
    "react": "15.4.2",
    "react-native": "^0.42.0",
    "react-native-config": "^0.1.2",
    "react-native-geocoder": "^0.4.6",
    "react-native-maps": "^0.13.0",
    "react-native-router-flux": "^3.38.0",
    "react-native-splash-screen": "^2.0.0",
    "react-native-svg": "^5.1.5",
    "victory-native": "^0.7.0"
  },
```
Here is my tide chart component that renders the line chart: https://gist.github.com/ryancoughlin/74e7d0909856e520943fb198c0282d59

Thoughts? I've been messing around with some items here, but can't seem to track down what is happening. 

![simulator screen shot mar 19 2017 8 37 46 pm](https://cloud.githubusercontent.com/assets/1182341/24086362/a3014e74-0ce4-11e7-90d8-d8554bcd3d44.png)

**Comment by boygirl** [comment]
**From @ryancoughlin on https://github.com/FormidableLabs/victory-native/pull/56**
I created similar JSON data with time and height keys over at http://beta.json-generator.com/ and I got the same results.

**Comment by ryancoughlin** [comment]
Oh good call, great thank you Lauren.

**Comment by matpaul** [comment]
Confirm, in my case line chart not rendering at all if i add animate property =( 
I demo app same behavior

**Comment by boygirl** [comment]
fixed in `0.7.2` but with an outstanding issue https://github.com/FormidableLabs/victory-native/issues/77
